traffic_monitor/cache/astats.go

```diff
@@ -116,8 +117,23 @@ func astatsParse(cacheName string, rdr io.Reader, pollCTX interface{}) (Statisti
 		astats.Ats["system.proc.loadavg"] = astats.System.ProcLoadavg
 		astats.Ats["system.proc.net.dev"] = astats.System.ProcNetDev
 
+		via := ctx.HTTPHeader.Get("Via")
+		if via != "" {
+			result := regexp.MustCompile(` ([a-z0-9\-]*)\..*comcast.net`).FindStringSubmatch(via)
+			if len(result) > 0 {
+				astats.Ats["via"] = result[1]
```

Comment:
Should this be `result[0]`?

Reply:
result[0] is the whole match, and result[1] is the selection.